packages/compass-crud/src/stores/crud-store.ts

```diff
@@ -1700,6 +1701,13 @@ class CrudStoreImpl
       }
     }
   }
+
+  openDeleteQueryExportToLanguageDialog(query: string): void {
+    this.localAppRegistry.emit('open-delete-query-export-to-language', query);
+    this.globalAppRegistry.emit('compass:export-to-language:opened', {
+      source: 'Bulk Delete',
+    });
```

Comment:
I don't see anything subscribed to this event, why are we adding this emit?

Reply:
The modal in the aggregation pipeline exporter is also doing this, so I expected it was some kind of metric. I can remove it, as the code doesn't depend on it.

Reply:
Ah, gotcha. I think this might be a leftover from when our analytics was event based, do you mind cleaning up both while you're at it?

Reply:
Sure, will do :D.